packages/sdk/src/algebraic_type.ts

```diff
@@ -377,7 +383,9 @@ export class AlgebraicType {
     return (
       this.isProductType() &&
       this.product.elements.length === 1 &&
-      this.product.elements[0].algebraicType.#isBytes() &&
+      // confusingly, Address and Identity's '__address_bytes' and '__identity_bytes' now store integers rather than byte arrays.
```

Comment:
That's why they're no longer called that...

Reply:
This method should be renamed, and likely made to accept the "wrapped" type as an argument rather than hardcoding it.